apps/api/src/server/applications/application/application.controller.js

```diff
@@ -19,7 +19,10 @@ const getServiceCustomerIds = (serviceCustomers) => {
 };
 
 /**
+ * Express request handler for creating application
+ *
  * @type {import('express').RequestHandler}
+ * @returns { Promise<void> }
```

Comment:
probably don't need this as the @type RequestHandler already contains the expected returned value.